### PR TITLE
Complete MkDocs documentation

### PR DESCRIPTION
**Pull request by alexandreroutier**

### CODE REVIEW

File: docs/Installation.md [hunk 1]
```diff
@@ -11,65 +11,51 @@ if you encounter any issues.
 
 ### Python environment
 You will need a Python environment to run Clinica. We advise you to
```
Comment (mdiazmel) [inline docs/Installation.md]:
```suggestion
You will need a Python environment to run ClinicaDL. We advise you to
```

**Review by mdiazmel — COMMENTED** [review]
Hi @alexandreroutier,

Thanks for this PR. 

These are some of the points to review before merging:

- In the installation page, I think that the developer installation should be deleted (someone contributing to this code will probably know how to do it). It will allow to have a simpler installation page, less overwhelming for the user.
- In several sections, we talk about `Running the pipeline` (see the subtitles of pages `Classify.md`, `Generate.md`, etc). This word is extracted from Clinica's vocabulary and, in my opinion, it doesn't reflects what is done in each steep (excepting for the "Preprocessing pipelines" section). The help instructions inside the code named this actions "tasks". Maybe is a better idea to change `Running the pipepile` by `Launching the task` .
- We should propose/indicate where to download the pre-trained models in the Classify section. I will prepare a commit to do this.

**Comment by alexandreroutier** [comment]
> * In the installation page, I think that the developer installation should be deleted (someone contributing to this code will probably know how to do it). It will allow to have a simpler installation page, less overwhelming for the user.

I used the material for Clinica documentation. With the removal of the Conda installation process for both Clinica & ClinicaDL, I agree with your suggestion :)

> * In several sections, we talk about `Running the pipeline` (see the subtitles of pages `Classify.md`, `Generate.md`, etc). This word is extracted from Clinica's vocabulary and, in my opinion, it doesn't reflects what is done in each steep (excepting for the "Preprocessing pipelines" section). The help instructions inside the code named this actions "tasks". Maybe is a better idea to change `Running the pipepile` by `Launching the task` .

Indeed. What about simply writing 'Running the task'? I don't have a strong opinion about it 🤔 

> * We should propose/indicate where to download the pre-trained models in the Classify section. I will prepare a commit to do this.

Great!

I am poking @14thibea regarding the remaining question.

File: docs/Installation.md [hunk 2]
```diff
@@ -38,7 +35,7 @@ conda activate clinicadlEnv
 pip install clinicadl
 ```
 
-### Running the ClinicaDL environment
+## Run the ClinicaDL environment
 #### Activation of the ClinicaDL environment
```
Comment (14thibea) [inline docs/Installation.md:39]: I would remove this title

File: docs/Installation.md [hunk 3]
```diff
@@ -81,14 +78,14 @@ conda activate clinicadlEnv
     ```
 
 
-#### Deactivation of the ClinicaDL environment
+### Deactivation of the ClinicaDL environment
```
Comment (14thibea) [inline docs/Installation.md:81]: and this one too

**Comment by mdiazmel** [comment]
I'll merge this one!

**Pull request merged**